### PR TITLE
Fixed #24102 - add name to unique_together in create_model

### PR DESCRIPTION
**Pull request by dnozay**

### CODE REVIEW

**Comment by timgraham** [comment]
buildbot, test this please.

**Comment by timgraham** [comment]
It seems this generates invalid SQL.

**Comment by dnozay** [comment]
yes, seems like sqlite does not like the syntax. I've updated the change for mysql backend where I know this would work.

can anybody ask buildbot to test a change?
buildbot, test this please.

**Comment by dnozay** [comment]
buildbot, test this please.

**Comment by dnozay** [comment]
@timgraham could you please ask buildbot to try again? thank you

File: django/db/backends/schema.py [hunk 1]
```diff
@@ -268,6 +268,9 @@ def create_model(self, model):
         for fields in model._meta.unique_together:
             columns = [model._meta.get_field(field).column for field in fields]
             column_sqls.append(self.sql_create_table_unique % {
+                # ticket #24102: constraint name may be useful for some backends
```
Comment (timgraham) [inline django/db/backends/schema.py]: I don't think there's a need for this comment, but a regression test would be useful.

Comment (dnozay) [inline django/db/backends/schema.py]: will see what I can cook up.

Comment (dnozay) [inline django/db/backends/schema.py]: @timgraham -  cannot add a regression test right now as `django.db.backends.sqlite3.introspection.get_constraints` does not return constraint names.

**Comment by dnozay** [comment]
@timgraham - did you see my comments by any chance?

**Comment by timgraham** [comment]
How about the other databases besides SQLite? We could skip this test there if necessary.

**Comment by timgraham** [comment]
Fixed in https://github.com/django/django/commit/01ec127baec38b7f8281e6eca9def40f22c9e5ad.

**Pull request closed without merging**